docs/debugging.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[debugging]]
+== Debugging the agent itself
+
+Hopefully the agent Just Works™ for you but depending on your situation the agent could need some tuning.
+
+Firstly, to know more about what's going on inside the agent, you can increase the amount of log messages it writes by setting the log level with the option `log_level = 0` -- `0` being the level of most messages, `DEBUG`.
+
+In your `config/elastic_apm.yml`:
+
+[source,yaml]
+----
+log_level: <%= Logger::DEBUG %>
+----
+
+[float]
+[[debugging-errors]]
+=== Errors
```

Comment:
(Thanks for updating the wording below.)

Following the same line of reasoning, I think using the title "Errors" gives the impression that this is definitely exceptional behaviour. Not sure what a better title is though. Maybe "Log Messages"?